### ISSUE TITLE
Block arguments /commands

### ISSUE THREAD

**Issue by Guest1131** [DoNotSpamPls/eZProtector#36]
Can you please add to the custom-commands, the possibility to block also the arguments of a command?

Example I want to block /rt

This example command has 2 arguments /rt add and /rt remove, I want /rt remove to be blocked by custom-commands, now I've tried but blocked the whole command together..not just the argument.

Can you add this option?

Thanks

**Comment by DoNotSpamPls** [comment]
After considering this feature request for a while, I have decided that it is out of scope - you would be better off setting permissions to deny /rt remove for players. 
Adding this feature to eZProtector would unfortunately cause more issues that it resolves
